test/image/mocks/gl2d_parcoords_1.json

```diff
@@ -0,0 +1,29 @@
+{
+  "layout": {
+    "width": 284,
+    "height": 400
+  },
+
+  "data": [{
+
+    "type": "parcoords",
```

Comment:
The `domain` computation now look fine :+1: 

But something weird happens to some axis ticks on some `restyle` calls e.g. `Plotly.restyle(gd, 'domain.y', [[0.2, 0.8]])`

![gifrecord_2017-02-09_120906](https://cloud.githubusercontent.com/assets/6675409/22794338/d2755412-eec0-11e6-8cc9-2d75c6beaea0.gif)